src/io/dsv.rs

```diff
@@ -30,10 +29,13 @@
 //! let table_reader: Box<dyn TableReader> = Box::new(csv_reader);
 //! ```
 //!
+//! The example first creates a [`DSVReader`] which uses comma as its separator.
+//! Then it packs it as a trait object of type [`TableReader`], which represents the visibility for the phyical layer.
+//!
 //! ## Physical layer
-//! The physical layer will only receive a trait object ['TableReader'][crate::io::TableReader], which contains
-//! the instantiated `csv_reader` from the above example.
-//! For this illustrative example the variable table_reader, as defined in the above example will be used.
+//! The physical layer receives a trait object [`TableReader`], containing
```

Comment:
```markdown
receives a [`TableReader`] trait object
````

Technically, it's also not containing, since it still _is_ the same object.